Formula/cht.sh.rb

```diff
@@ -0,0 +1,36 @@
+class ChtSh < Formula
+  desc "Command-line client for cheat.sh"
+  homepage "https://cheat.sh/"
+  url "https://github.com/chubin/cheat.sh.git",
```

Comment:
Formulae in the core repository must have a stable version tagged by the upstream project.
> https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
So if I understand you correctly having version baked in code (https://github.com/chubin/cheat.sh/blob/c4c3a62/share/cht.sh.txt#L27) isn't enough because there isn't a git tag? If so I'll ask upstream to see if tagging is an option.

Reply:
It would be great that upstream can tag the release, the homebrew distribution can benefit from it a lot.